doc/guide.rst

```diff
@@ -0,0 +1,179 @@
+.. _guide_label:
+
+Guide to Simpleline
+===================
+
+Simpleline is the text user interface framework completely written in Python 3 with a possibility to
+have :ref:`non-python event loop <event_loops_label>`. With exception of optional event loops
+Simpleline doesn't have almost any dependency to external libraries.
```

Comment:
`doesn't have almost any dependency to external libraries.` -> `has almost no dependency on external libraries.`